### ISSUE TITLE
Glow + Levitation creeper effects

### ISSUE THREAD

**Issue by QuentinTy** [bitquest/bitquest#185]
in the code, bitquest/bitquest/src/main/java/com/bitquest/bitquest/entityevents.java there is a chance for GLOWING and LEVITATION. glowing might not be much of an issue however if a player were to get a levitation creeper to spawn and it blows up someone without a compass then the account could be stuck until help was received by a mod. (on second look glowing is unobtainable to players and only shows in offworld places)

**Comment by theraccoonbear** [comment]
@QuentinTy I'm not sure I follow.  What would be the problem here?

**Comment by syrianrue** [comment]
well lets assume a levitation creeper did explode near you, maybe while floating away. you would have infinite levitation so you would just constantly keep floating up to the sky. using a compass/eye to get back down would just restart you at ground level before floating back up.

https://minecraft.gamepedia.com/Status_effect#Levitation

**Comment by theraccoonbear** [comment]
Interesting, I did not know that was a thing

**Comment by QuentinTy** [comment]
Issue, newer players may not have compasses and this issue could be very annoying to mods

**Comment by u8iuui11** [comment]
Got hit by one of those creepers. Had to get my friend to give me a bucket of milk making me lose all of my other buffs =/ .

**Comment by Shiny-andKittens** [comment]
@QuentinTy can you show me where those effects are? I can try to fix it

**Comment by QuentinTy** [comment]
Yeah, I sent you message on slack with screenshot as well as a link to the bug can be found (here)[https://github.com/bitquest/bitquest/blob/master/src/main/java/com/bitquest/bitquest/EntityEvents.java]. Line 640ish

**Comment by Shiny-andKittens** [comment]
@explodi I fixed this in a new PR#190
